cli_reference/get_started_cli.adoc

```diff
@@ -19,8 +19,8 @@ logging in to create your first project.
 
 ifdef::atomic-registry[]
 [NOTE]
-Some sections of this document refer to OpenShift. {product-title} is based on
-OpenShift technology.
+Some sections of this document refer to {product-title}. {product-title} is
+based on {product-title} technology.
```

Comment:
Ha! Yeah, that's quite a sentence and it can probably stand to be cut out!  :no_good:   :laughing: